libsentrykube/tests/conftest.py

```diff
@@ -20,3 +23,105 @@ def set_workspaceroot() -> Iterator[None]:
     os.environ["SENTRY_KUBE_CONFIG_FILE"] = str(workspace_root() / "config.yaml")
     yield
     set_workspace_root_start(start_workspace_root)
+
+
+CLUSTER_1 = {
+    "id": "cluster1",
+    "services": [
+        "k8s/services/my_service",
+        "k8s/services/another_service",
+    ],
+    "my_service": {"key1": "value1"},
+}
+
+CLUSTER_2 = {
+    "id": "cluster2",
+    "services": [
+        "k8s/services/my_service",
+    ],
+}
+
+CONFIGURATION = {
+    "sites": {
+        "test_site": {
+            "name": "us",
+            "region": "us-central1",
+            "zone": "b",
+            "network": "global/networks/sentry",
+            "subnetwork": "regions/us-central1/subnetworks/sentry-default",
+        }
+    },
+    "silo_regions": {
+        "customer1": {
+            "bastion": {
+                "spawner_endpoint": "FIXME",
+                "site": "test_site",
+            },
+            "k8s": {
+                "root": "k8s",
+                "cluster_def_root": "clusters",
+                "services_in_cluster_config": "True",
+                "materialized_manifests": "materialized_manifests",
+            },
+        },
+        "customer2": {
+            "bastion": {
+                "spawner_endpoint": "FIXME",
+                "site": "test_site",
+            },
+            "k8s": {
+                "root": "k8s",
+                "cluster_def_root": "clusters",
+                "services_in_cluster_config": "True",
+                "materialized_manifests": "materialized_manifests",
+            },
+        },
+    },
+}
```

Comment:
Are all of these fields needed? I'm not sure what the point of `os.environ["SENTRY_KUBE_CONFIG_FILE"] = str(
        workspace_root() / "cli_config/configuration.yaml"
    )` is

Reply:
> os.environ["SENTRY_KUBE_CONFIG_FILE"] = str( workspace_root() / "cli_config/configuration.yaml" )

Without setting `SENTRY_KUBE_CONFIG_FILE` the system will try to load the prod configuration in the ops repo.

> Are all of these fields needed?

Pretty much all of that. We can skip customer2 for this specific test scenario, but I would like the basic config to be reusable for other tests. Basically the config library valdidates that all fields are there no matter if you use them or not.